contracts/sfc/SFC.sol

```diff
@@ -459,9 +459,12 @@ contract SFC is OwnableUpgradeable, UUPSUpgradeable, Version {
         emit TreasuryFeesResolved(fees);
     }
 
-    /// burnFTM allows SFC to burn an arbitrary amount of FTM tokens.
-    function burnFTM(uint256 amount) external onlyOwner {
-        _burnFTM(amount);
+    /// Burn native tokens by sending them to the SFC contract.
+    function burnNativeTokens() external payable {
+        if (msg.value == 0) {
+            revert ZeroAmount();
+        }
+        _burnNativeTokens(msg.value);
```

Comment:
`_burnNativeTokens()` lowers `totalSupply`, but this function doesn't increase the `totalSupply`.

So if you send a lot of `FTM` to `burnNativeTokens()` you can arbitrarily reduce `totalSupply`.